tests/test_suites/LongSystemTests/test_sqlite_stress_several_goals.sh

```diff
@@ -0,0 +1,56 @@
+timeout_set 15 minutes
+
+sqlite_test_path=$(realpath test_utils/sqlite_stress_test.py)
+
+CHUNKSERVERS=5 \
+	USE_RAMDISK=YES \
+	MOUNT_EXTRA_CONFIG="sfscachemode=NEVER" \
+	CHUNKSERVER_EXTRA_CONFIG="MAGIC_DEBUG_LOG=${TEMP_DIR}/log" \
```

Comment:
Please, consider that sometimes is needed to specify the expected log level we expect through the `LOG_FLUSH_ON` option:

```suggestion
	CHUNKSERVER_EXTRA_CONFIG="MAGIC_DEBUG_LOG=${TEMP_DIR}/log | LOG_FLUSH_ON=DEBUG" \
```

Reply:
As I'm only for errors, and there should be quite plenty if some issue occurs, I think this should not be necessary.